variables.tf

```diff
@@ -1,4 +1,76 @@
 variable "github_token" {
   type        = string
   description = "Github token"
-}
\ No newline at end of file
+}
+
+variable "github_organization" {
+  type        = string
+  description = "Github organization"
+  default     = "one-beyond"
+
+}
+
+variable "github_users" {
+  type = list(object({
+    username          = string
+    organization_role = string
+    memberships       = list(string)
+    teams = list(object({
+      team      = string
+      team_role = string
+    }))
+  }))
+  default = [
```

Comment:
Are we adding only these 4 users to test?

Reply:
If, after testing that these 4 users are correctly added to the state when we apply all the changes, I will still import all the users